Editor/UI/Drawers/TabsGroupDrawer.cs

```diff
@@ -0,0 +1,65 @@
+using System;
+using System.Linq;
+using Innoactive.Creator.Core.Tabs;
+using UnityEditor;
+using UnityEngine;
+
+namespace Innoactive.CreatorEditor.UI.Drawers
+{
+    [DefaultTrainingDrawer(typeof(ITabsGroup))]
+    internal class TabsGroupDrawer : AbstractDrawer
```

Comment:
Documentation would be nice

Reply:
internal class which is a drawer, would except people to understand without docu :P